### PR TITLE
OSDOCS-8387 Porting cloud experts rosaworkshop.io Lab Intro

### PR DESCRIPTION
**Pull request by xJustin**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-8387

Ported from: https://www.rosaworkshop.io/ostoy/lab_intro/ 
Link to docs preview:

https://69245--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+:context: cloud-experts-deploying-application-intro
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+== Introduction
+
+After successfully provisioning your cluster, you can deploy an application on it. This application will allow us to become more familiar with some of the features of {product-title} and Kubernetes.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:14]:
```suggestion
After successfully provisioning your cluster, you can deploy an application on it. This application allows you to become more familiar with some of the features of {product-title} (ROSA) and Kubernetes.
```
I've been changing the first-person plurals when I can to fit more with our normal docs.

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+=== Lab overview
+In this lab, you will complete the following set of tasks designed to help you understand the concepts of deploying and operating container-based applications:
+
+* Deploy a Node.js based app by using S2I and Kubernetes Deployment objects.
+* Set up a continuous delivery (CD) pipeline to automatically push source code changes.
+* Explore logging.
+* Experience self healing of applications.
+* Explore configuration management through configmaps, secrets and environment variables.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:23]:
```suggestion
* Explore configuration management through configmaps, secrets, and environment variables.
```

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Use persistent storage to share data across pod restarts.
+* Explore networking within Kubernetes and applications.
+* Familiarize yourself with {product-title} and Kubernetes functionality.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:26]:
```suggestion
* Familiarize yourself with ROSA and Kubernetes functionality.
```
The {product-title} spells out ROSA, which we only do once. So after that, we just use ROSA.

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* Automatically scale pods based on load from the Horizontal Pod Autoscaler.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:27]:
```suggestion
* Automatically scale pods based on loads from the Horizontal Pod Autoscaler.
```
Alternatively, "the load" would also work, depending on context.

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* Use AWS Controllers for Kubernetes (ACK) to deploy and use an S3 bucket.
+
+You will conduct this lab with the {product-title} CLI. You can also use the {product-title} web console to complete these tasks.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:30]:
```suggestion
You will conduct this lab with either the ROSA CLI or the ROSA web user interface (UI). 
```

**Review by jneczypor — COMMENTED** [review]
Looks good to me. Just a few minor changes :). Thanks!

**Comment by xJustin** [comment]
Thanks @jneczypor ! I incorporated all of your suggestions.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* Explore logging.
+* Experience self healing of applications.
+* Explore configuration management through configmaps, secrets, and environment variables.
+* Use persistent storage to share data across pod restarts.
+* Explore networking within Kubernetes and applications.
+* Familiarize yourself with ROSA and Kubernetes functionality.
+* Automatically scale pods based on loads from the Horizontal Pod Autoscaler.
+* Use AWS Controllers for Kubernetes (ACK) to deploy and use an S3 bucket.
+
+You will conduct this lab with either the ROSA CLI or the ROSA web user interface (UI).
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:30]: I keep bouncing this sentence in my head, and I cannot tell why I don't like it...

Maybe "This workshop lab module (or whatever) uses either the ROSA CLI or the {cluster-manager-url} ROSA UI"?

Comment (xJustin) [reply cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc:30]: Yeah, I definitely had issues with this one as well... I will mull it over a bit before we merge this.

**Pull request merged**